spec.bs

```diff
@@ -1721,8 +1680,13 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
           1. Let |directFromSellerSignalsForBuyer| be the result of running
             [=get direct from seller signals for a buyer=] with |directFromSellerSignals|, and |ig|'s
             [=interest group/owner=].
+          1. Let |dataVersion| be |trustedBiddingSignalsBatcher|'s [=trusted bidding signals batcher/data versions=]
+             [|ig|'s [=interest group/name=]].
+          1. If |dataVersion| is not null, then [=map/set=] |browserSignals|["{{BiddingBrowserSignals/dataVersion}}"] to |dataVersion|.

```

Comment:
nit: I think we're wrapping to 100 columns -- this is getting a bit long even for my monitor for side-by-side diff.

Reply:
@JensenPaul @morlovich Looks like this merged -- could we fix in a follow-up?

Reply:
Also, did @domfarolino sign off?

Reply:
Well, conveniently I just mailed a follow up for your review (for other reasons).
I probably still had some ultra-long stuff in there, though, because some of the identifiers are really long making it hard to wrap without being silly (I already have some unneedes spaces in foo [bar] )